x-pack/plugins/alerting/server/task_runner/create_execution_handler.test.ts

```diff
@@ -69,15 +73,55 @@ test('calls executeAction per selected action', async () => {
     `);
 });
 
-test('limits executeAction per action group', async () => {
+test(`doesn't call actionsPlugin.execute for disabled actionTypes`, async () => {
+  createExecutionHandlerParams.actionsPlugin.isActionTypeEnabled.mockReturnValueOnce(false);
+  createExecutionHandlerParams.actionsPlugin.isActionTypeEnabled.mockReturnValueOnce(true);
```

Comment:
This doesn't seem right.  I think we want the `false` version and not the `true` version, and I'd expect the the `await executionHandler()` call to throw.

Reply:
> This doesn't seem right. I think we want the false version and not the true version

These are doubled to mock the result of nth call. Below there are two `actions` and the test makes sure the first action (within `createExecutionHandlerParams.actions`) doesn't get passed to `actionsPlugin.execute(...)` and only the second one.

> I'd expect the the await executionHandler() call to throw.

I'm thinking this comment is about a test scenario and not a feature of the executionHandler? Happy to add a test for this.

Reply:
Ah!  I see now - yes, it was a question about the test, wasn't obvious to me that `mockReturnValueOnce()` calls get "stacked up", and missed that there were two actions there as well!  Worth a comment.

Reply:
I can see how it's not clear, I went ahead and added a comment https://github.com/elastic/kibana/pull/59699/commits/b9c41f3d06b8886b0bc7d36955533539aeeae9ee